plugins/config/config.go

```diff
@@ -0,0 +1,30 @@
+package config
+
+import (
+	"fmt"
+	"strings"
+
+	common "github.com/dokku/dokku/plugins/common"
+)
+
+func GetWithDefault(appName string, key string, defaultValue string) string {
```

Comment:
Only app config. We can have an alternative method for retrieving the global config version of any value.